frontend/.env

```diff
@@ -1,3 +1,4 @@
 REACT_APP_API_BASE=http://localhost:8000
 REACT_APP_GOOGLE_CLIENT_ID=SET_THIS
 REACT_APP_JIRA_URL=https://jira.server
+REACT_APP_MAX_NON_BILLABLE_TO_BILLABLE_RATIO=.25
```

Comment:
This should be 0.4 with the new Non-billade/Billable metric.
`1 h (Non-billable) / 2.5 h (Billable) = 0.4` as mentioned [on the handbook](https://handbook.opencraft.com/en/latest/cell_budgets/)

Reply:
Thank you!